draft-ietf-httpbis-unprompted-auth.md

```diff
@@ -201,6 +203,11 @@ Key ID:
 
 : The key ID sent in the `k` Parameter (see {{parameter-k}}).
 
+Public Key:
+
+: The public key used by the server to validate the signature provided by the
+client.
```

Comment:
What is the encoding of this public key? We probably need to be clear about that in order for this to work.

Reply:
Good point. Fixed in 9dc88eef6543e47dd32c84427a1df73424e4edff